tests/hpxml_translator_test.rb

```diff
@@ -105,6 +105,84 @@ def test_invalid
     end
   end
 
+  def test_generalized_hvac
+    # single-speed air conditioner
+    cOOL_EIR_FT_SPEC = [[-3.302695861, 0.137871531, -0.001056996, -0.012573945, 0.000214638, -0.000145054]]
+    seer_to_expected_eer = { 13 => 11.2, 14 => 12.1, 15 => 13.0, 16 => 13.9 }
+    seer_to_expected_eer.each do |seer, expected_eer|
+      # fan_power_rated = HVAC.get_fan_power_rated(seer)
+      fan_power_rated = 0.365
+      actual_eer = HVAC.calc_EER_cooling_1spd(seer, fan_power_rated, cOOL_EIR_FT_SPEC)
+      assert_in_epsilon(expected_eer, actual_eer, 0.01)
+    end
+
+    # single-speed air source heat pump
+    cOOL_EIR_FT_SPEC = [[-3.437356399, 0.136656369, -0.001049231, -0.0079378, 0.000185435, -0.0001441]]
+    seer_to_expected_eer = { 13 => 11.31, 14 => 12.21, 15 => 13.12 }
+    seer_to_expected_eer.each do |seer, expected_eer|
+      fan_power_rated = 0.365
+      # fan_power_rated = HVAC.get_fan_power_rated(seer)
+      actual_eer = HVAC.calc_EER_cooling_1spd(seer, fan_power_rated, cOOL_EIR_FT_SPEC)
+      assert_in_epsilon(expected_eer, actual_eer, 0.01)
+    end
+    hEAT_EIR_FT_SPEC = [[0.718398423, 0.003498178, 0.000142202, -0.005724331, 0.00014085, -0.000215321]]
+    hEAT_CAP_FT_SPEC = [[0.566333415, -0.000744164, -0.0000103, 0.009414634, 0.0000506, -0.00000675]]
+    hspf_to_expected_cop = { 7.7 => 3.09, 8.2 => 3.35, 8.5 => 3.51 }
+    hspf_to_expected_cop.each do |hspf, expected_cop|
+      fan_power_rated = 0.365
+      actual_cop = HVAC.calc_COP_heating_1spd(hspf, Constants.C_d, fan_power_rated, hEAT_EIR_FT_SPEC, hEAT_CAP_FT_SPEC)
+      assert_in_epsilon(expected_cop, actual_cop, 0.01)
+    end
+
+    # two-speed air conditioner
+    cOOL_EIR_FT_SPEC = [[-3.877526888, 0.164566276, -0.001272755, -0.019956043, 0.000256512, -0.000133539],
+                        [-1.990708931, 0.093969249, -0.00073335, -0.009062553, 0.000165099, -0.0000997]]
+    cOOL_CAP_FT_SPEC = [[3.940185508, -0.104723455, 0.001019298, 0.006471171, -0.00000953, -0.000161658],
+                        [3.109456535, -0.085520461, 0.000863238, 0.00863049, -0.0000210, -0.000140186]]
+    capacity_ratios = [0.72, 1.0]
+    fan_speed_ratios = [0.86, 1.0]
```

Comment:
Can you wrap all of these assumed values into hvac.rb methods that are referenced here and in hvac.rb? That way we only have the values defined once and they can't become out of sync.

Reply:
Not sure why you resolved this comment, it still hasn't been addressed.

Reply:
Not sure what is left to be addressed...? You're talking about wrapping `capacity_ratios` and `fan_speed_ratios`, and not just the curves?

Reply:
My thinking was that we shouldn't be setting these values both here and somewhere else, since they can become out of sync. But now that I think about it, I guess I don't know whether that's a problem. For example, if we decided to change the capacity ratios we use in the model, maybe the tests could/would stay unchanged? If that's the case, then my comment doesn't apply.